modeltranslation/fields.py

```diff
@@ -400,6 +447,27 @@ def __get__(self, instance, owner):
         return None
 
 
+class TranslatedManyToManyDescriptor(object):
```

Comment:
```suggestion
class TranslatedManyToManyDescriptor:
```

We don't need this `object` in python3.